dubbo-registry/dubbo-registry-consul/pom.xml

```diff
@@ -36,6 +36,12 @@
             <groupId>com.ecwid.consul</groupId>
             <artifactId>consul-api</artifactId>
         </dependency>
+        <dependency>
+            <groupId>com.pszymczyk.consul</groupId>
+            <artifactId>embedded-consul</artifactId>
+            <version>1.1.0</version>
```

Comment:
You've specified 2.0.0 in dubbo-dependencies-bom/pom.xml, why it is 1.1.0 here?

Reply:
sorry for the mistake.